### PR TITLE
Error Handle Update - path_finder

### PR DESCRIPTION
**Pull request by NikkiBytes**
Added unique error catches for path finder module. 
Now catches for different types of errors and uses logging to track cases.

### CODE REVIEW

File: src/utils/metakg/path_finder.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import networkx as nx
 import logging
+
+import networkx as nx
+
 from controller.metakg import MetaKG
 from model import ConsolidatedMetaKGDoc
 
-logger=logging.basicConfig(level=logging.INFO, filename="missing_bte.log")
+logging.basicConfig(level="INFO")
```
Comment (newgene) [inline src/utils/metakg/path_finder.py:8]: I think we should remove this line here. Typically we don't want to set it to a fixed value in our code.

File: src/admin.py [hunk 2]
```diff
@@ -36,8 +36,6 @@
 from model import ConsolidatedMetaKGDoc, MetaKGDoc
 from utils import indices
 
-logging.basicConfig(level="INFO")
-
```
Comment (newgene) [inline src/admin.py:40]: @NikkiBytes although we said we should remove this line in another file, let's still keep this line in this `admin.py` file. These are admin tasks we need to run as developers, it's probably always a good idea to turn it on.

**Review by newgene — COMMENTED** [review]
@NikkiBytes I left only one minor comment, after it's addressed, this PR is good to merge.

File: src/admin.py (hunk 2 above)
Comment (NikkiBytes) [reply src/admin.py]: Hey I added it [here](https://github.com/SmartAPI/smartAPI/blob/851ee5c16d16ab57af3eed361ed4e8257e4db0f5/src/admin.py#L39) to `admin.py`.

File: src/admin.py [hunk 3]
```diff
@@ -38,7 +38,6 @@
 
 logging.basicConfig(level="INFO")
 
-
```
[the remark this answers is not in the document]
Comment (NikkiBytes) [reply src/admin.py:41]: oh yeah I made the change through github directly, so I will flake8 this now!

**Pull request merged**